tagged-manifest.xml

```diff
@@ -3,5 +3,5 @@
   <!-- incremental version bump: droid-src-sony-aosp-10/1.12.2 -->
   <remote fetch="https://github.com/mer-hybris" name="hybris"/>
   <include name="tagged-localbuild.xml"/>
-  <project name="droid-src-sony" path="rpm" remote="hybris" revision="da04953176810bba60b422e1acf6cb8365131adb"/>
+  <project name="droid-src-sony" path="rpm" remote="hybris" revision="58b8d1660bc99235d0bbe2ae64e2debac017eddd"/>
```

Comment:
Please change this PR to draft to prevent accidental merging, before the depending [PR](https://github.com/mer-hybris/droid-src-sony/pull/26/commits) is merged:) This hash will need updating to a merge commit.

Reply:
Please review the droid-src PR then :)